### ISSUE TITLE
Raspberry Pi4 support

### ISSUE THREAD

**Issue by pascalbrunner** [jovandeginste/workout-tracker#72]
Hi, 

I tried the docker ony my RPI4 but always get this:
The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested

Is there a simple fix?

The project looks really nice and I would love to use it :)

Thank you!

**Comment by jovandeginste** [comment]
You could try with the non-Docker arm64 release; I have no experience with building arm containers, so I'll need to look into that...

**Comment by jovandeginste** [comment]
Can you test with the new image (should be available in a few minutes)?

**Comment by pascalbrunner** [comment]
Thanks for your fast reply!
I tried:
docker run -p 8080:8080 ghcr.io/jovandeginste/workout-tracker:docker-arm

And I am able to access the welcome page on my RPI4 🎉.
If I run into any troubles I would update here but it seems your fix is working for me! Thanks!

**Comment by jovandeginste** [comment]
Wow, so it works? You should also be able to use the master branch now.

**Comment by pascalbrunner** [comment]
Yes it works! I just also tried the new master branch! Thanks for fixing that quickly :)
And you can now add to the Git that RPI4 is also supported 😜

Big thanks again and great work! The import with gpx files and visualization is great :)
